plugin/editorconfig.vim

```diff
@@ -227,6 +227,12 @@ function! s:UseConfigFiles() abort " Apply config to the current buffer {{{1
             \ ' on file "' . l:buffer_name . '"'
     endif
 
+    if exists("b:EditorConfig_disable")
+        if b:EditorConfig_disable
```

Comment:
nit: this could be done on fewer lines:

    if exists("b:EditorConfig_disable") && b:EditorConfig_disable
        return
    endif

I checked it myself on 8.1 - `:echo exists('b:NOPE') && b:NOPE` does not give an error, even though `:echo b:NOPE` does.  (Short-circuiting: see [`:help expr3`](http://vimdoc.sourceforge.net/htmldoc/eval.html#expr3)).